webpack.config.js

```diff
@@ -81,7 +99,11 @@ const webpackConfig = {
 		rules: [
 			{
 				test: /\.jsx?$/,
-				exclude: /node_modules[\/\\](?!notifications-panel)/,
+				// Should Babel transpile the file at `filepath`?
+				include: ( filepath ) => {
+					return ( ! pathIsInside( filepath, nodeModulesDir ) ||
+						hasPkgEsnext( filepath ) );
+				},
```

Comment:
this one feels oddly verbose - any reason not to simplify the syntax?

```js
include: path => ! pathIsInside( path, nodeModulesDir ) || hasPkgEsnext( path ),
```

Reply:
148ebc7